### DISCUSSION TITLE
Cannot connect to device on port COM8

### DISCUSSION THREAD

**Discussion by chadoulis** [STELLA-Landsat/STELLA#24] [Build Notes: How do I make a STELLA?]
When clicking on the Serial button in Mu, to look at the output of the STELLA instrument, I get the following `Cannot connect to device on port COM8` error:

![Screenshot 2023-06-11 220428](https://github.com/STELLA-Landsat/STELLA/assets/25674261/265a3b2f-d3a9-449f-873a-fc6b59b0a77c)

Any ideas on how I can fix this?

**Comment by paulmirel** [comment]
That error can mean that the COM port is in use. The two things we use to connect to the STELLA are the Dataviewer web page, and the Mu editor. They don't like it when the other one is already connected to the STELLA. Try unplugging, plugging back in, and connecting again.
